packages/nodes-base/nodes/Kafka/KafkaTrigger.node.ts

```diff
@@ -141,7 +148,7 @@ export class KafkaTrigger implements INodeType {
 
 		await consumer.connect();
 
-		await consumer.subscribe({ topic, fromBeginning: true });
+		await consumer.subscribe({ topic, fromBeginning: (options.fromBeginning)? true : false });
```

Comment:
Please make sure, that the `options` variable is assigned before using one of its fields.